types/errors/callbacks.go

```diff
@@ -0,0 +1,15 @@
+package errors
```

Comment:
Why creating a new file for a single function?

Reply:
Because it's content doesn't match to other files content.